### PR TITLE
docs(guides): change var to let and move lodash under dependencies

### PR DESCRIPTION
**Pull request by sudarsangp**

### CODE REVIEW

File: src/content/guides/getting-started.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ __src/index.js__
 + import _ from 'lodash';
 +
   function component() {
-    var element = document.createElement('div');
+    let element = document.createElement('div');
 
 -   // Lodash, currently included via a script, is required for this line to work
 +   // Lodash, now imported by this script
```
Comment (Legends) [inline src/content/guides/getting-started.md]: Remove this comment completely.

**Comment by montogeek** [comment]
@sudarsangp Please sign CLA>

**Comment by sudarsangp** [comment]
@montogeek signed already
image

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**